setup.py

```diff
@@ -114,10 +114,10 @@ def get_version():
 
             # write the version file
             if os.path.exists(BUILDRUNNER_DIR):
-                with open(VERSION_FILE, 'w', encoding='utf8') as fobj:
-                    fobj.write("__version__ = '%s'\n" % new_version)
+                with open(VERSION_FILE, 'w', encoding='utf8') as _fobj:
```

Comment:
Why was the `_` added?

Reply:
pylint complained about it because it is defined in the outer scope.